index.js

```diff
@@ -217,8 +217,11 @@ const sleepingTime = sleepingTimeInMinutes * 60000;
         } catch (e) {
             console.log('Routine error at: ', new Date().toLocaleString(), e)
         }
+        const response = await fetch('https://api.steemmonsters.io/battle/history?player=' + process.env.ACCOUNT);
+        const battleResult = await response.json();
+        await console.log("winner:", battleResult.battles[0].winner)
```

Comment:
I like the functionality, but not how it is implemented.
First because it makes an additional call that it's not really needed since the it can be read by the UI.
Second because if the battle didn't happen for any reason, this will print the winner of the previous battle and will lead to confusion.

Reply:
I agree with this, you can read in the UI along with the details of ECR and DEC won